### PR TITLE
Bartlomiej biel/loda onepage

### PR DESCRIPTION
**Pull request by Bielbiel**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
-    "name": "theamforest-exercises",
-    "version": "0.0.0",
-    "dependencies": {
-        "bootstrap": "3.3.7"
-    }
+  "name": "theamforest-exercises",
+  "version": "0.0.0",
+  "dependencies": {
+    "bootstrap": "3.3.7",
+    "isotope": "^3.0.1",
```
Comment (Nigatsu) [inline bower.json]: don't use ^ or * in version

File: index.html [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Loda Onepage Personal Portfolio Template</title>
+    <link rel="stylesheet" href="app/bower_components/bootstrap/dist/css/bootstrap.css">
+    <link rel="stylesheet" href="project.css">
+</head>
+<body>
+<div class="margin">
+    <div class="container">
+        <!-- Static navbar -->
+        <nav class="navbar navbar-fixed-top navbar-default margin" style="background-color: rgba(255, 255, 255, 1);">
```
Comment (Nigatsu) [inline index.html]: Don't use style in html, move to proper class

File: index.html [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+                            data-target="#navbar"
+                            aria-expanded="false" aria-controls="navbar">
+                        <span class="sr-only">Toggle navigation</span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                    </button>
+                    <a class="hederSpan" href="index.html">Loda</a>
+                </div>
+                <div id="navbar" class="navbar-collapse collapse" style="float: right">
```
Comment (Nigatsu) [inline index.html]: style

File: index.html [hunk 4]
```diff
@@ -0,0 +1,363 @@
...
+                            <div id="1" class="item brand ">
+                                <div class="item-wrap" >
+                                    <a class="image-modal">
+                                        <img src="photos/picture1.png" alt="">
+                                    </a>
+                                </div>
+                            </div>
+                            <div id="2" class="item web brand">
+                                <div class="item-wrap">
+                                    <a class="image-modal">
```
Comment (Nigatsu) [inline index.html]: This is not wrong but having block element in inline element looks wrong

**Review by aniaw — COMMENTED** [review]
1. You should change the structure of the project. *app* folder with only *bower_components* folder inside is not necessary. Move your project files (index.html, function.js, project.css ) into *app* folder

2. Look at font on original page - you should use the same font.

3. You forget about social icons on the bottom of the page (facebook, twitter, instagram etc.)

4. You have to fix spacing between elements.

**Pull request closed without merging**